### DISCUSSION TITLE
CUSA14285 - Odallus The Dark Call

### DISCUSSION THREAD

**Discussion by Zehkarluz** [GoldHEN/GoldHEN_Cheat_Repository#12] [Cheat requests]
Infinite health, magic and money, please. Thanks

**Comment by kmeps4** [comment]
Will check if there's a cheat available to add

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
